worldedit-core/src/main/java/com/sk89q/worldedit/registry/Category.java

```diff
@@ -36,7 +36,7 @@ protected Category(final String id) {
         this.id = id;
     }
 
-    public final String getId() {
+    public final String id() {
```

Comment:
This is not an overridden method, so that's breaking I guess?

Reply:
Upstream had it implementing Keyed, fixed